### ISSUE TITLE
[BUG - Static Site] Cannot navigate to touchpoints modal button with keyboard

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#6364]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When tabbing through the static site page, a user cannot navigate to the "How can we improve SimpleReport" touchpoints modal button in the bottom right hand corner.

<img width="1512" alt="Screenshot 2023-08-18 at 16 57 23" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/7aa45608-40a8-4130-8d2a-24b27697fb63">

In addition, when a user navigates via keyboard to "Skip to feedback" link, their focus is not actually moved to the "How can we improve SimpleReport" touchpoints modal button in the bottom right hand corner. (For comparison, see behavior when "Skip to main content" is entered.
<img width="1512" alt="Screenshot 2023-08-18 at 16 54 18" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/1fd26ec9-39fd-4081-8f2a-66f46e862b15">

### Steps To Reproduce

1. On the static site, keyboard tab through any page and try to get to the "How can we improve SimpleReport" button.
2. Tab to the "Skip to feedback" link.
3. See that steps 1 and 2 results in no change in focus to the "How can we improve SimpleReport" button.

### Expected Behavior

Doing steps 1 and 2 above results in a change of focus to the "How can we improve SimpleReport" button.

### Does this issue have a priority label?

- [X] I have added a priority label

### Has issue been assigned to a team?

- [X] I have added a team label
